tests/test_pseudocone.py

```diff
@@ -145,8 +239,32 @@ def test_list_test_data_users_missing_params(mock_db_data, pseudocone_server):
 
 
 @patch("app.pseudocone.DatabaseClient.load_data", return_value=SINGLE_USER_SINGLE_ITEM_MULTI_INTERACTION)
-def test_list_test_data_users_multi_interaction2(mock_db_data, pseudocone_server):
+def test_list_test_data_users_between_dates_missing_params(mock_db_data, pseudocone_server):
+    users = [pseudocone_pb2.UserParam(id="1"), pseudocone_pb2.UserParam(id="2")]
+
+    request_without_start = pseudocone_pb2.ListTestDataUsersBetweenDatesRequest(limit=1,
+                                                                                users=users,
+                                                                                end_interaction_time="P0Y1M7DT0H0M0S")
 
+    request_without_end = \
+        pseudocone_pb2.ListTestDataUsersBetweenDatesRequest(limit=1, users=users,
+                                                            start_interaction_time="2018-02-01T00:00:26.318497Z")
+
+    request_without_start_or_end = \
+        pseudocone_pb2.ListTestDataUsersBetweenDatesRequest(limit=1, users=users)
+
+    with pytest.raises(Exception):
+        pseudocone_server["stub"].ListTestDataUsers(request=request_without_start)
+
+    with pytest.raises(Exception):
+        pseudocone_server["stub"].ListTestDataUsers(request=request_without_end)
+
+    with pytest.raises(Exception):
+        pseudocone_server["stub"].ListTestDataUsers(request=request_without_start_or_end)
```

Comment:
I think this is the same as the test with `request_without_start` because we first hit the start_interaction_time Exception - could perhaps remove.